source/code-snippets/usage-examples/deleteOne.js

```diff
@@ -1,26 +1,37 @@
+/* Delete a document */
+
+// Import the MongoClient type from the mongodb package.
 import { MongoClient } from "mongodb";
 
 // Replace the uri string with your MongoDB deployment's connection string.
 const uri = "<connection string uri>";
 
+// Create a new client and connect to MongoDB.
 const client = new MongoClient(uri);
 
 async function run() {
   try {
+    // Access the movies collection from the sample_mflix database.
     const database = client.db("sample_mflix");
     const movies = database.collection("movies");
 
-    // Query for a movie that has title "Annie Hall"
+    // Create a filter for movies with a title that is "Annie Hall".
```

Comment:
Suggestion:
I think it would be good to omit this since it is self-explanatory.
This comment also uses the term "filter" which doesn't match the term used earlier in the content "query document". I think it would be better to match the terminology used in the content.

Applies to all instances.